### PR TITLE
Add ArkUrl copy to clipboard in resource-view

### PR DESCRIPTION
**Pull request by flavens**
closes [DSP-441](https://dasch.myjetbrains.com/youtrack/issue/DSP-441)

### CODE REVIEW

File: projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.html [hunk 1]
```diff
@@ -1,5 +1,21 @@
 <div class="resource-view" *ngIf="resource">
 
+    <!-- citation link - ARK URL -->
+    <div class="ark-url-label">
+        <label for="clipboard-arkurl">Citation link (ARK URL)</label>
```
Comment (SepidehAlassi) [inline projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.html:5]: Capital "L" in link

File: projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.ts [hunk 2]
```diff
@@ -73,10 +80,28 @@ export class ResourceViewComponent implements OnChanges {
...
   }
 
+  /**
+   * Display message to confirm the copy of the citation link (ARK URL)
+   * @param message
+   * @param action
+   */
+  openSnackBar(message: string, action: string) {
+    message = 'Copied to clipboard!';
+    action = 'Citation link';
```
Comment (SepidehAlassi) [inline projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.ts:99]: Capital "L" in link

File: projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.html (hunk 1 above)
Comment (flavens) [reply projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.html]: fixed in b3c521b

File: projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.ts (hunk 2 above)
Comment (flavens) [reply projects/dsp-ui/src/lib/viewer/views/resource-view/resource-view.component.ts]: fixed in b3c521b

**Pull request merged**